app/controllers/catalog_controller.rb

```diff
@@ -265,24 +265,6 @@ def x_show
     tree_select
   end
 
-  # Tree node selected in explorer
-  def tree_select
-    @explorer   = true
-    @lastaction = "explorer"
-    self.x_active_tree = params[:tree] if params[:tree]
```

Comment:
@martinpovolny i can see this setting to be in use when clicking on a link to a catalog item from outside catalog explorer.